modules/stateful/main.tf

```diff
@@ -19,6 +19,13 @@ resource "aws_volume_attachment" "default" {
   device_name = "/dev/xvdh"
   volume_id   = "${element(aws_ebs_volume.default.*.id, count.index)}"
   instance_id = "${element(aws_instance.default.*.id, count.index)}"
+
+  # There's a circular dependency here on destroy with the instance. In order
+  # to dettach, you must first unmount. We assume that the attachment would only
+  # be destroyed when the instance is also being destoryed. In that case, we can
```

Comment:
s/destoryed/destroyed/

Reply:
@adborden for future fix, but not enough to block the merge.

Reply:
Thanks, opened https://github.com/GSA/datagov-infrastructure-modules/pull/36. This is one of my most common typos 😄